### PR TITLE
Sell services on ob

### PR DESCRIPTION
**Pull request by jenncloud**

### CODE REVIEW

File: _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: "How to <span class='post-title-extra'>Sell Services for Cryptocurrency</span> on OpenBazaar"
+layout: post
+date: '2019-04-25 08:30:00 -0600'
+social_title: 'How To Sell Services for Cryptocurrency on OpenBazaar'
+social_description: 'On OpenBazaar you can easily sell services for cryptocurrency. Here’s how.'
+---
+
+Making sales is about more than just moving physical products. People also make entire careers—and often very high level careers!—out of selling the services they perform. These can be things done from anywhere in the world like like web design, software development and copywriting, or very local things like medical assistance, legal representation and lawn care.
```
Comment (jjeffryes) [inline _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown]: Are those hyphen (-) or em-dashes(—)? They should be em-dashes if you're using them to break apart a sentence.

File: _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown [hunk 2]
```diff
@@ -1,46 +1,50 @@
...
 
-In fact, services make up [percent] of the worldwide economy.
+In fact, [services make up almost 70% of value added to the world's economy](https://www2.deloitte.com/insights/us/en/economy/issues-by-the-numbers/trade-in-services-economy-growth.html). This kind of market share shows an enormous opportunity!
 
-Sell Your Services Online
+### Sell Your Services Online
 
-OpenBazaar’s unique approach to an escrow account makes it even easier for a provider to list and bill their services. By setting up a moderated transaction a provider can be sure that their client has the funds for contract they are providing and upon successful completion of the work the funds can simply be released to the provider.
+OpenBazaar’s unique approach to an escrow account makes it even easier for a provider to list and bill their services. By setting up a moderated transaction a service seller can be sure that their client has the funds for contract they are agreeing to and upon successful completion of the work the funds can simply be released to the seller.
```
Comment (SamPatt) [inline _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown]: Even easier than what?

File: _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown [hunk 3]
```diff
@@ -1,46 +1,50 @@
...
 
 On OpenBazaar you can easily sell services for cryptocurrency. Here’s how.
 
 #### 1. Define your service
 
-What services do you sell? How do you usually price them? Many providers offer services by the hour or on a custom basis, but to list services for sale on OpenBazaar it’s easiest to start by “productizing” them, or structuring them to be constrained to a set cost. For example, if you provide language translation services you may create listings with different prices for 100 - 200 words translated, 300 - 500 words translated and 500 - 750 words translated.
+What services do you sell? How do you usually price them? Many providers offer services by the hour or on a custom basis, but to list services for sale on OpenBazaar it’s easiest to start by “productizing” them, or structuring them to be constrained to a set cost. For example, if you provide language translation services you may create 3 different products with different prices. For example:
+
+1. 100 - 200 words translated
```
Comment (SamPatt) [inline _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown]: 200-300 seems to be missing.

File: _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown (hunk 1 above)
Comment (jenncloud) [reply _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown]: They're totally emdashes but they seem so small coming through VS Code??

File: _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown (hunk 2 above)
Comment (jenncloud) [reply _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown]: Noted, just going with "easy!"

File: _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+With the **return policy** you can set this up however you’d like. You may want to say no refunds or perhaps offer a 100% satisfaction guarantee with a full refund option. 
+
+**Terms & conditions** can be added if you have any conditions or advisories you want to make people aware of when accessing or using your digital good.
+
+Select the **coins** you’d like to receive as payment: [Bitcoin](https://bitcoin.org), [Bitcoin Cash](https://bitcoincash.org), [Litecoin](https://litecoin.org) and/or [Zcash](https://z.cash). it can be just one of them or any combination.
+
+
+#### 3. Share your services from OpenBazaar.com 
+
+OpenBazaar is a marketplace where people can browse and discover the items that you have for sale—but that’s not the only way they can find them! [OpenBazaar.com](https://openbazaar.com) exists to make it easy for you and others to share stores and listings to your social media and other communities.
```
Comment (jjeffryes) [inline _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown]: em dash here too

Comment (jenncloud) [reply _posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown]: I swear they are em dashes!! Agh!

**Pull request merged**